source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -64,6 +70,18 @@ This procedure assumes the host machine has ``kubectl`` installed and
 configured with access to the target Kubernetes cluster. The host machine 
 *must* have access to a web browser application.
 
+.. cond:: openshift
```

Comment:
So I have omitted three pre-reqs from our legacy docs:

- Manually creating a namespace for the Tenant
- Manually creating credentials for tenants
- Manually creating SecurityContexts

@cniackz and I will go over this next week to confirm that these are all unnecessary in OpenShift w.r.t. installing Operator + deploying a tenant.